### PR TITLE
Add webhook management

### PR DESCRIPTION
**Pull request by fivetran-aleksandrboldyrev**

### CODE REVIEW

File: fivetran/data_source_webhook.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+				Computed:    true,
+				Description: "The webhook creation timestamp",
+			},
+			"created_by": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The ID of the user who created the webhook.",
+			},
+		},
+	}
```
Comment (beevital) [inline fivetran/data_source_webhook.go:63]: Schema could be unified with resource and datasource will be implemented in ~10 lines of code. Check out how dbt-resources implemented.

File: fivetran/data_source_webhook.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	for k, v := range msi {
+		if err := d.Set(k, v); err != nil {
+			return newDiagAppend(diags, diag.Error, "set error", fmt.Sprint(err))
+		}
+	}
+
+	d.SetId(resp.Data.Id)
+
+	return diags
+}
```
Comment (beevital) [inline fivetran/data_source_webhook.go:97]: This method is almost identical with resource read method. It also could be unified.

File: fivetran/resource_webhook.go [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+			"events": {
+				Type:        schema.TypeSet,
+				Required:    true,
+				Description: "The array of event types",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"active": {
+				Type:        schema.TypeBool,
+				Required:    true,
+				Description: "Boolean, if set to true, webhooks are immediately sent in response to events",
```
Comment (beevital) [inline fivetran/resource_webhook.go:51]: tfplugindocs utility automatically adds field type
```suggestion
				Description: "If set to true, webhooks are immediately sent in response to events",
```

File: fivetran/resource_webhook.go [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+			svcAcc.Events(xInterfaceStrXStr(v.(*schema.Set).List()))
+		}
+
+		resp, err := svcAcc.Do(ctx)
+		if err != nil {
+			return newDiagAppend(diags, diag.Error, "create error", fmt.Sprintf("%v; code: %v", err, resp.Code))
+		}
+
+		d.SetId(resp.Data.Id)
+	} else if d.Get("type").(string) == "group" && d.Get("group_id").(string) != "" {
```
Comment (beevital) [inline fivetran/resource_webhook.go:99]: Maybe let's extract two methods? One for group, one for account.

**Comment by fivetran-aleksandrboldyrev** [comment]
Merge commit - it's about apply suggestions and merge it with local changes (not with main branch)

**Review by beevital — APPROVED** [review]
LGTM

**Pull request merged**